src/webview/SearchSidebar/SearchResultList/index.tsx

```diff
@@ -3,17 +3,17 @@ import { DisplayResult } from '../../../types'
 import TreeItem from './comps/TreeItem'
 
 interface SearchResultListProps {
-  matches: Array<[string, DisplayResult[]]>
+  matches: Map<string, DisplayResult[]>
 }
 
-const SearchResultList = ({ matches }: SearchResultListProps) => {
+const SearchResultList = memo(({ matches }: SearchResultListProps) => {
   return (
     <div style={{ flexGrow: '1', overflowY: 'scroll' }}>
-      {matches.map(([filePath, match]) => {
+      {Array.from(matches.entries()).map(([filePath, match]) => {
         return <TreeItem filePath={filePath} matches={match} key={filePath} />
       })}
     </div>
   )
-}
+})
 
 export default memo(SearchResultList)
```

Comment:
SearchResultList has already been memorized here

Reply:
Nice catch! I'm removing it now.